src/js/utils/CFHostAPI/CFHostAPI.js

```diff
@@ -123,14 +149,14 @@ export function fullZoneSet({ user_key, zone_name }, onSuccess, onError) {
         }
     };
 
-    return send('POST', opts, onSuccess, onError);
+    return send('POST', opts, callback);
 }
 
-function send(method, opts, onSuccess, onError) {
+function send(method, opts, callback) {
     if (method.toUpperCase() === 'GET') {
         opts.parameters.host_key = hostKey;
     } else {
         opts.body.host_key = hostKey;
     }
-    return http.request(method, ENDPOINT, opts, onSuccess, onError);
+    return http.request(method, ENDPOINT, opts, hostCallback(callback));
```

Comment:
why is this one `hostCallback(callback)` where as the others are like `send('POST', opts, callback);` with wrapping with `hostCallback` function